lesson-03/task-01.js

```diff
@@ -0,0 +1,123 @@
+// =========================================
+// Task-01
+// =========================================
+
+const user = {
+  name: "Mango",
+  age: 20,
+  hobby: "html",
+  premium: true,
+};
+
+user.mood = "happy";
+user.hobby = "skydiving";
+user.premium = false;
+
+for (let key of Object.keys(user)) {
+  console.log(`${key}: ${user[key]}`);
+}
+
+// ========================================
+// Task-02
+// =========================================
+
+const countProps = function (obj) {
+  return Object.entries(obj).length;
+};
+
+console.log(countProps({ mail: "poly@mail.com", isOnline: true, score: 500 }));
+
+// =========================================
+// Task-03
+// =========================================
+
+const findBestEmployee = function (employee) {
+  let count = 0;
+  const result = [];
+  for (let value of Object.values(employee)) {
+    if (value > count) {
+      count = value;
+    }
+  }
+  Object.keys(employee).forEach((key) => {
+    if (employee[key] === count) {
+      result.push(key);
+    }
+  });
+
+  return result.join("");
```

Comment:
якщо в масиві тільки одне значення, то можна його отримувати по індексу 
```suggestion
  return result[0];
```